pycaret/classification.py

```diff
@@ -2341,7 +2403,7 @@ def load_model(
     )
 
 
-def automl(optimize: str = "Accuracy", use_holdout: bool = False) -> Any:
+def automl(optimize: str = "Accuracy", use_holdout: bool = False, return_train_score: bool = False) -> Any:
```

Comment:
has this been formatted with black?

Reply:
@Yard1 I have not formatted the code with black, will do it and then push again.

Reply:
Should I format all files with black? Or this particular file only, because after I applied black in the root folder, 38 files were affected. Please let me know @Yard1

Reply:
Only the files that you modified.

Reply:
Okay, sure!

Reply:
Done!